src/main.js

```diff
@@ -10,6 +10,4 @@ createDivCard.innerHTML+=pokemon.name + "<br>" + pokemon.numero
 getDivCards.appendChild(createDivCard)
 let createImgPokemon=document.createElement("img")
 createDivCard.appendChild(createImgPokemon)
-createImgPokemon.srcset=pokemon.imagem
-
-}
```

Comment:
Camis, vc acabou removendo a chave de fechamento do for, aí deu erro na hora de rodar o código.
Precisa colocar de volta *_*